engine/context/aci-integration.md

```diff
@@ -1,465 +1,5 @@
 ---
 title: Deploying Docker containers on Azure
 description: Deploying Docker containers on Azure
-keywords: Docker, Azure, Integration, ACI, context, Compose, cli, deploy, containers, cloud
-toc_min: 1
-toc_max: 2
+redirect_to: /cloud/aci-integration/

```

Comment:
Perhaps better to add a `redirect_from` in the new location, so that we don't have to keep this file (same for the other files)

Reply:
Also; could this be done with a _move_ of the file, so that we keep git history? (i.e. same commit that moves the file and adds the redirect)?

Reply:
Indeed, much better! fixed and re-organised commits